lib/helpers/calculator_helpers.js

```diff
@@ -268,17 +268,17 @@ module.exports = class CalculatorHelpers {
             dataElements.forEach((dataElement) => {
               if (dataElement.id != null) {
                 // if an episode has already been created set the result for the population to 1
-                if (episodeResults[dataElement.id]) {
-                  episodeResults[dataElement.id][popCode] = 1;
+                if (episodeResults[dataElement.id.value]) {
```

Comment:
@serhii-ilin can this be adjusted in typescript models? we can discuss if I'm not clear here

Reply:
Not sure what you mean..

Reply:
maybe we'll discuss it in my next PR. I'll merge this so that Mike would be able to create patient for further testing